### PR TITLE
Bump up wpewebkit version to 2.38.1

### PR DESCRIPTION
**Pull request by neodesys**

### CODE REVIEW

File: recipes/wpebackend-fdo.recipe [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 class Recipe(recipe.Recipe):
     name = 'wpebackend-fdo'
-    version = '1.8.0'
-    stype = SourceType.GIT
+    version = '1.14.0'
+    stype = SourceType.TARBALL
     btype = BuildType.CMAKE
-    remotes = {'origin': 'https://github.com/Igalia/WPEBackend-fdo.git'}
-    commit = '5523f39d46d398608544a501c39b6f1c3d41233f'
+    url = 'https://wpewebkit.org/releases/wpebackend-fdo-{0}.tar.xz'.format(version)
```
Comment (zhani) [inline recipes/wpebackend-fdo.recipe:11]: not sure why we have this recipe. We are not using this

File: recipes/wpewebkit/0008-Call-WPEProcessProvider-kill-on-ProcessLauncher-plat.patch [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+diff --git a/Source/WebKit/UIProcess/Launcher/glib/ProcessLauncherGLib.cpp b/Source/WebKit/UIProcess/Launcher/glib/ProcessLauncherGLib.cpp
+index 83a683b6..67742150 100644
+--- a/Source/WebKit/UIProcess/Launcher/glib/ProcessLauncherGLib.cpp
++++ b/Source/WebKit/UIProcess/Launcher/glib/ProcessLauncherGLib.cpp
+@@ -247,6 +247,12 @@ void ProcessLauncher::terminateProcess()
+ 
+ void ProcessLauncher::platformInvalidate()
+ {
++#if defined(WPE_ENABLE_PROCESS) && WPE_ENABLE_PROCESS
```
Comment (zhani) [inline recipes/wpewebkit/0008-Call-WPEProcessProvider-kill-on-ProcessLauncher-plat.patch:29]: This patch was removed. platformInvalidate() should be left as it was (ie. empty function)

**Review by zhani — CHANGES_REQUESTED** [review]

File: recipes/wpebackend-fdo.recipe (hunk 1 above)
Comment (neodesys) [reply recipes/wpebackend-fdo.recipe:11]: :+1: removed.

File: recipes/wpewebkit/0008-Call-WPEProcessProvider-kill-on-ProcessLauncher-plat.patch (hunk 2 above)
Comment (neodesys) [reply recipes/wpewebkit/0008-Call-WPEProcessProvider-kill-on-ProcessLauncher-plat.patch]: Done :+1:

File: recipes/wpewebkit.recipe [hunk 3]
```diff
@@ -30,25 +28,23 @@ class Recipe(recipe.Recipe):
...
                         -DUSE_GSTREAMER_GL=ON \
                         -DUSE_GSTREAMER_NATIVE_VIDEO=ON \
                         -DUSE_GSTREAMER_NATIVE_AUDIO=ON \
                         -DUSE_WOFF2=OFF \
-                        -DUSE_SYSTEMD=OFF \
+                        -DENABLE_DOCUMENTATION=OFF \
+                        -DENABLE_INTROSPECTION=OFF \
+                        -DENABLE_JOURNALD_LOG=OFF \
                         -DENABLE_BUBBLEWRAP_SANDBOX=OFF \
-                        -DWPE_ENABLE_PROCESS=1 \
```
Comment (zhani) [inline recipes/wpewebkit.recipe:42]: why did you remove this? Now patch 0006-Enable-libwpe-process-management-api.patch has no effect anymore

**Review by zhani — CHANGES_REQUESTED** [review]

File: recipes/wpewebkit.recipe (hunk 3 above)
Comment (neodesys) [reply recipes/wpewebkit.recipe:42]: I didn't want to remove it, this is an error. This is why it is not working with wpe-android. I fixed it :+1:

**Review by zhani — APPROVED** [review]
